Twig/Auth0Extension.php

```diff
@@ -41,10 +41,29 @@ public function getFunctions()
     {
         return [
             new \Twig_SimpleFunction('state_parameter', [$this, 'stateParameter']),
+            new \Twig_SimpleFunction('sso_params', [$this, 'ssoParams']),
             new \Twig_SimpleFunction('sso_login_url', [$this, 'loginUrl']),
         ];
     }
 
+    public function ssoParams()
+    {
+        if ($this->csrfTokenManager) {
+            $csrfToken = $this->csrfTokenManager->getToken('auth0-sso');
+
+            $csrf_token = $csrfToken->getValue();
+            $domain = $this->generator->getAuth0Domain();
```

Comment:
... and add them as constructor arguments in the extension. You'll also need to add them to the service definition in Resouces/config/services.yml.